apps/meteor/server/services/messages/hooks/BeforeSaveBadWords.ts

```diff
@@ -24,13 +27,20 @@ export class BeforeSaveBadWords {
 
 		this.badWords = new Filter(options);
 
+		try {
+			this.badWordsRegex = new RegExp(`(?<=^|[\\p{P}\\p{Z}])(${badWords.join('|')})(?=$|[\\p{P}\\p{Z}])`, 'gmiu');
+		} catch (error) {
+			this.badWordsRegex = null;
```

Comment:
silently consuming the error. can we log this pls?

Reply:
added the logs